src/components/Patient/PatientHome.tsx

```
@@ -680,7 +680,671 @@ export const PatientHome = (props: {
+                        </div>
+                        <div className="sm:col-span-1">
+                          <dt
+                            title=" Origin facility"
+                            className="flex items-center text-sm font-semibold leading-5 text-zinc-400"
+                          >
+                            <CareIcon
+                              icon="l-plane-fly"
+                              className="mr-2 text-lg"
+                            />
+                            <dd className="text-sm font-bold leading-5 text-secondary-900">
+                              {(shift.origin_facility_object || {})?.name}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)